docs/enhancements/day2-master.md

```diff
@@ -0,0 +1,37 @@
+---
+title: day2-master
+authors:
+- "@oamizur"
+  creation-date: 2022-07-10
+  last-updated: 2022-07-24
```

Comment:
nit: indentation error (`*-date` should not be tabulated)

Reply:
Changed